packages/component-model-manuscript/entities/manuscript/index.test.js

```diff
@@ -154,14 +154,13 @@ describe('Manuscript', () => {
 
     describe('given there is a single file', () => {
       let manuscript
-      let file
```

Comment:
removed the `file/s` objects from these types of tests - to enforce referencing them directly from the `manuscript` object.